extras/git-hooks/post-receive

```diff
@@ -97,7 +81,7 @@ for line in lines:
 
     try:
         base = repo.revparse_single(base)
-        ancestors = list(gather_ancestors(base))
+        ancestors = [commit.id for commit in repo.walk(base.id)]
```

Comment:
I take it back.  The list of ancestors can be in any order.  We're only checking for membership in it to figure out when to stop second part of the hook later on.